nxdrive/client/local/linux.py

```diff
@@ -24,8 +26,19 @@ class LocalClient(LocalClientMixin):
 
     def has_folder_icon(self, ref: Path) -> bool:
         """Check if the folder icon is set."""
-        # To be implementation with https://jira.nuxeo.com/browse/NXDRIVE-1831
-        return True
+        matcher = re.compile(r"metadata::emblems: \[nuxeo.*\]")
```

Comment:
The whole method could be simplified by check cheking the presence of `~/.local/share/icons/emblem-nuxeo.svg`, WDYT?

Reply:
If ~/.local/share/icons/emblem-nuxeo.svg exists but not the attribute on the Drive folder then the emblem wont't show. In my opinion we shoul add the file verification before the regexp and return False if the file doesn't exist.